core/modules/catalog/store/product/actions.ts

```diff
@@ -25,6 +25,8 @@ import RootState from '@vue-storefront/core/types/RootState'
 import ProductState from '../../types/ProductState'
 import { Logger } from '@vue-storefront/core/lib/logger';
 import { TaskQueue } from '@vue-storefront/core/lib/sync'
+import { isServer } from '@vue-storefront/core/helpers'
```

Comment:
```suggestion
```

@pkarw build is not passing, because isServer is already imported in line 4 here